### ISSUE TITLE
[telegram] notify when chain sync is lost

### ISSUE THREAD

**Issue by niteshbalusu11** [alexbosworth/balanceofsatoshis#566]
I recently lost chain sync on my node a few times with my pruned node. It would be nice if telegram can notify if LND loses chain sync for more than X mins.

**Comment by alexbosworth** [comment]
yeah it sounds good, there is already some logic to monitor that in the limit forwarding command I think?

**Comment by niteshbalusu11** [comment]
Idk. I can check if we already have it somewhere.
